From a1902b8d9e1a98aa2271d1f441470c11b2594e48 Mon Sep 17 00:00:00 2001
From: Neel Chauhan <neel@neelc.org>
Date: Fri, 24 May 2024 12:26:06 -0400
Subject: [PATCH] increase default max password length to 128

Ported to AXP.OS lineage-22.2 (rhodep fork) from GrapheneOS 4131f09b20ea.

Change-Id: Iccc3616880251afc4c722d012b8fca784137c26b
---
 core/java/android/app/admin/DevicePolicyManager.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/java/android/app/admin/DevicePolicyManager.java b/core/java/android/app/admin/DevicePolicyManager.java
index 5d3842b89..ff83fa71a 100644
--- a/core/java/android/app/admin/DevicePolicyManager.java
+++ b/core/java/android/app/admin/DevicePolicyManager.java
@@ -3738,7 +3738,7 @@ public class DevicePolicyManager {
      * Maximum supported password length. Kind-of arbitrary.
      * @hide
      */
-    public static final int MAX_PASSWORD_LENGTH = 64;
+    public static final int MAX_PASSWORD_LENGTH = 128;
 
     /**
      * Service Action: Service implemented by a device owner or profile owner supervision app to
-- 
2.43.0

